lib/nrf_modem_lib/nrf_modem_lib_trace.c

```diff
@@ -274,5 +279,17 @@ int nrf_modem_lib_trace_stop(void)
 	return 0;
 }
 
+void nrf_modem_lib_trace_deinit(void)
+{
+	is_transport_initialized = false;
+
+#ifdef CONFIG_NRF_MODEM_LIB_TRACE_MEDIUM_UART
+	nrfx_uarte_uninit(&uarte_inst);
+#endif
+#ifdef CONFIG_NRF_MODEM_LIB_TRACE_MEDIUM_RTT
+	/* Drain data? */
```

Comment:
```suggestion
	SEGGER_RTT_Init();
```

Reply:
I looked for one such API but I could not find it either, but I am not convinced we should init Segger here. The peripheral is re-initialized on `nrf_modem_lib_trace_init()` already, this is a de-initialization function. The comment seems appropriate, since flushing any pending writes seems more likely what we would want to do in the future.

Reply:
The problem is that we only write a comment here and not any code that shows how to drain data. And we also have a question mark that indicates uncertainty. You may chose leave it as is, as I no longer have a strong opinion on this.

Reply:
I agree, ideally we should have had code here. The reason why we don't is that we don't quite have time to write it now :) I have reworded the comment slightly.